online-shop/src/app/services/common.ts

```diff
@@ -0,0 +1 @@
+export const API_BASE_URL = 'http://localhost:3000';
```

Comment:
**suggestion:** you cand keep this base url inside the `environment.ts` file or you could use a [proxy](https://angular.io/guide/build#proxying-to-a-backend-server)